### ISSUE TITLE
Question for the  deduplicate_bismark

### ISSUE THREAD

**Issue by hmyh1202** [FelixKrueger/Bismark#465]
Hi, Felix:

    If the order of fastq reads affect reads record of the deduped bam ？
    I have a fastq file and split it into ten parts and run bismark in parallel to get the aligned bam, and merge that bam files into a big bam by using samtools merge. And then get the deduped bam file using deduplicate_bismark. If the deduplicate_bismark only hold the read with higtest map quality ?  Or only retain the first duplicated read ? If so, the deduped bam would recored random reads record for duplicate reads of original fastq. 
 
The best ！

**Comment by FelixKrueger** [comment]
Yes, `deduplicate_bismark` will keep the first read to a given position, and does not apply additional filtering for highest mapping quality or most consistent methylation call etc. In our minds, this is near enough keeping a random read.

Just as a comment, `deduplicate_bismark` also has the option `--multiple` which will take a number of BAM files and treat them as a single input. For this, it internalle merges the files using `samtools cat` (you should not use `samtools merge` for paired-end files as it can occasionally disturb the R1-R2 order on subsequent lines).

**Comment by hmyh1202** [comment]
Okay, got it, thank you !
